app/cdap/components/FetchIconsFromWidget/utils.tsx

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import {
+  fetchConnectionDetails,
+  fetchConnectors,
+  getCategoriesToConnectorsMap,
+} from 'components/Connections/Create/reducer';
+import DataPrepStore from 'components/DataPrep/store';
+import DataPrepActions from 'components/DataPrep/store/DataPrepActions';
+import {
+  IConnectorArray,
+  IConnectorDetailPayloadArray,
+  IConnectorTypes,
+} from 'components/FetchIconsFromWidget/types';
+import WidgetSVG from 'components/FetchIconsFromWidget/Widget';
+import { ImportDatasetIcon } from 'components/WrangleHome/Components/WrangleCard/iconStore/ImportDatasetIcon';
```

Comment:
make ImportDatasetIcon as camelCase as this is a variable name
if it is a component then name it as TitleCase

Reply:
This is a variable name but it holds react SVG element. So, not making any changes to the casing.

Reply:
Done